79-notebook-v7/notebook-v7.md

```diff
@@ -1,19 +1,42 @@
-issue-number: NN
-pr-number: NN
+issue-number: 78
+pr-number: 79
 date-started: 2021-11-12
 ---
 
-**Note on the status of this document:** this is a very early draft, still not ready even for public vote. It supersedes the existing two open issues [#6210](https://github.com/jupyter/notebook/issues/6210) and [#6220](https://github.com/jupyter/notebook/issues/6220), we aim to collect input and feedback so we can rapidly identify any major remaining concerns and focus energy on implementing this plan. We note that so far feedback on the main plan has been very positive, so we expect to move forward with the main plan, but want to identify specific details and issues that may have been overlooked, to ensure the smoothest transition possible.
-
-
 # Jupyter Notebook version 7
 
+## Summary
+
 This JEP presents a path forward for the evolution of the Jupyter Notebook application in a way that is technically consistent with the rest of our tools and thus more sustainable, while meeting the needs of our users.
 
-This document proposes that the next major release of the [Jupyter Notebook](https://jupyter-notebook.readthedocs.io) application, version 7, will be based on the JupyterLab codebase, but will provide an equivalent user experience to the current (version 6) application.  Jupyter Notebook version 7 ("Notebook v7" henceforth) will achieve this by using the current version of [RetroLab](https://github.com/jupyterlab/retrolab). The following three key ideas are central to this planned transition:
+This document proposes that the next major release of the [Jupyter Notebook](https://jupyter-notebook.readthedocs.io) application, version 7, will be based on the JupyterLab codebase, but will provide an equivalent user experience to the current (version 6) application.  Jupyter Notebook version 7 ("Notebook v7" henceforth) will achieve this by using the current version of [RetroLab](https://github.com/jupyterlab/retrolab).
+
+
+## Motivation
+
+The classic Jupyter notebook interface is currently maintained by a skeleton crew of developers. On the other hand, many people still use the classic interface today.
+In short, we have a widely-used piece of software, but that is outdated and with very little developer support.
+
+There are a few major challenges with the current Notebook implementation:
+
+- Most people currently "maintaining" this repository are not frontend developers. When issues are raised in the Javascript APIs, the team lacks the experience to review/respond to these issues.
+- The Javascript is a outdated, so even frontend developers don't want to engage these issues.
+- The repository still receives a steady stream of issues and PRs, even though the project is technically in maintenance mode, meaning we'll only review bug/security fixes.
+
+
+The urgency of this issue (i.e. what to do with classic notebook) comes from a growing fear/possibility that notebook will break permanently because of decaying dependencies.
```

Comment:
```suggestion
The urgency of this issue (i.e. what to do with classic notebook) comes in order to protect the notebook from permanently breaking because of decaying dependencies while still offering our user community a modern Jupyter Notebook that has the familiar user experience they prefer.
```